brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-814 Modified title for date field in content Timeline Block

### Description
Title for date field improved for content timeline block

### Screenshots


### Types of changes
Improvement

### How has this been tested?
- Add a content timeline block
- Select a single timeline child block and check date field title

### Checklist:
